src/shared/api/genomeNexusClientInstance.ts

```diff
@@ -8,7 +8,7 @@ async function checkVersion(client:GenomeNexusAPI) {
     }
 }
 
-const client = new GenomeNexusAPI(getGenomeNexusApiUrl());
+const client = new GenomeNexusAPI("https://genomenexus-pr-221.herokuapp.com/");
```

Comment:
I guess this is a temporary change to make the pr work, let's not forget to change it back before merging.